src/app/series/series.routing.ts

```diff
@@ -3,22 +3,42 @@ import { Routes, RouterModule } from '@angular/router';
 import { AuthGuard } from '../shared';
 
 import { SeriesComponent } from './series.component';
+import { SeriesBasicComponent } from './directives/series-basic.component';
+import { SeriesImageComponent } from './directives/series-image.component';
+import { SeriesTemplatesComponent } from './directives/series-templates.component';
+import { SeriesAdvancedComponent } from './directives/series-advanced.component';
 
 export const seriesRoutes: Routes = [
   {
-    path: 'series/create',
+    path: 'series/:id',
     component: SeriesComponent,
-    canActivate: [AuthGuard]
+    canActivate: [AuthGuard],
+    children: [
+      { path: '',          component: SeriesBasicComponent },
+      { path: 'image',     component: SeriesImageComponent },
+      { path: 'templates', component: SeriesTemplatesComponent },
+      { path: 'advanced',  component: SeriesAdvancedComponent }
+    ]
   },
   {
-    path: 'series/:id',
+    path: 'series/new',
```

Comment:
Good point - I hadn't noticed if order-of-routes mattered.  Will double check that.

I do like having an explicit `series/new` route - so I don't have to check if `params.id === 'new'` when the component subscribes to the route params.  Maybe I can just DRY this up by putting the child routes in a shared const or something.